speechbrain/core.py

```diff
@@ -266,6 +266,12 @@ def parse_arguments(arg_list=None):
         type=int,
         help="Number of optimizer steps to run. If not passed, all epochs are run.",
     )
+    parser.add_argument(
```

Comment:
Hum, this must be discussed internally, but I don't see any reason to have this. Stopping with respect to optimisation steps, ok, but not training time.

Reply:
Will remove this later